packages/paste-website/src/pages/getting-started/inclusive-design/index.mdx

```diff
@@ -0,0 +1,285 @@
+      </ListItem>
+      <ListItem>
+        Provide enough spacing between lines of text (`line-height`){' '}
+        <Anchor href="https://www.w3.org/TR/WCAG20-TECHS/C21.html">(Why it matters)</Anchor>
+      </ListItem>
+      <ListItem>
+        Avoid all-caps text{' '}
+        <Anchor href="https://github.com/humanmade/hm-pattern-library/issues/75">(Why it matters)</Anchor>
+      </ListItem>
+      <ListItem>
+        Ensure that content is written as clearly and simply as possible{' '}
+        <Anchor href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-supplements.html">(Why it matters)</Anchor>
+      </ListItem>
+      <ListItem>Provide descriptive captions for figures</ListItem>
+      <ListItem>Warn users of links that have unusual behaviors, like linking off-site, or loading a new tab</ListItem>
+      <ListItem>Use textual labels to make voice activation cues obvious</ListItem>
+      <ListItem>Do not mark up subheadings/straplines with separate heading elements</ListItem>
+      <ListItem>Ensure primary calls to action are easy to recognize and reach</ListItem>
+      <ListItem>
+        Avoid images of text — text that cannot be translated, selected, or understood by assistive tech
+      </ListItem>
+      <ListItem>Use well-established, therefore recognizable, icons and symbols</ListItem>
+      <ListItem>Begin long, multi-section documents with a table of contents</ListItem>
+      <ListItem>Don"t make users perform actions to reveal content unless completely necessary</ListItem>
+      <ListItem>If content is meant to be hidden, ensure it is properly hidden to all users</ListItem>
+      <ListItem>Do not auto focus form fields, on page load</ListItem>
+      <ListItem>Break up long and complex forms into discrete sections and/or screens</ListItem>
+      <ListItem>
+        Make forms as short as possible; offer shortcuts like autocompleting the address using the postcode
+      </ListItem>
+      <ListItem>Inform the user when there are important changes to the application state</ListItem>
+      <ListItem>
+        Make sure the purpose of a link is clearly described: "read more" vs. "read more about accessibility"
+      </ListItem>
+    </UnorderedList>
+  </DisclosureContent>
+</Disclosure>
+
+<Disclosure variant="contained">
+  <DisclosureHeading as="h3" variant="heading30">
+    For UX and front-end engineers
+  </DisclosureHeading>
+  <DisclosureContent>
+    <UnorderedList>
+      <ListItem>
+        Minify CSS and JS, and remove unused/redundant code{' '}
+        <Anchor href="https://developers.google.com/speed/docs/insights/MinifyResources">(Why it matters)</Anchor>
+      </ListItem>
+      <ListItem>
+        Maintain terse HTML, without over-reliance on `div` scaffolding{' '}
+        <Anchor href="http://designingforperformance.com/optimizing-markup-and-styles/#divitis">
+          (Why it matters)
+        </Anchor>
+      </ListItem>
```

Comment:
```suggestion
```
Let's remove this. Divitis is kind of outdated these days and is often used in a not entirely healthy way to shame folks. Due to the nature of component encapsulation, reuse and composition, there will be alot of frequent cases you could call out as "divitis" but the benefits vastly out weigh the potential negatives.